UnitySDK/Assets/ML-Agents/Editor/BuilderUtils.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Linq;
+using UnityEditor;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+using System.IO;
+
+namespace MLAgents
+{
+	public class BuilderUtils
+	{
+		public static void SwitchAllLearningBrainToControlMode()
+		{
+			Debug.Log("The Switching to control mode function is triggered");
+			string[] scenePaths = Directory.GetFiles("Assets/ML-Agents/Examples/", "*.unity", SearchOption.AllDirectories);
+
+			foreach (string scenePath in scenePaths)
+			{
+				var curScene = EditorSceneManager.OpenScene(scenePath);
+				var aca = SceneAsset.FindObjectOfType<Academy>();
+				if (aca != null)
+				{
+					var learningBrains = aca.broadcastHub.broadcastingBrains.Where(
+						x => x != null && x is LearningBrain);
+		
+					foreach (Brain brain in learningBrains)
+					{
+						if (!aca.broadcastHub.IsControlled(brain))
+						{
+							Debug.Log("Switched brain in scene " + scenePath);
+							aca.broadcastHub._brainsToControl.Add(brain);
```

Comment:
Please use `SetControlled(brain, true)` instead of modifying the private property _brainsToControl

Reply:
Will change.